setup.py

```diff
@@ -69,3 +56,5 @@ def _get_version() -> str:
     python_requires='>=3.5',
     install_requires=["PyYAML"],
 )
+
+version_backup.rename(version_path)
```

Comment:
If `setup()` call crashes, the project would end "broken" (as it's interrupted between one rename and the other). Maybe you could wrap this in a try/finally or something?

Reply:
I changed it to be in a 'finally'